snap/snapcraft.yaml

```diff
@@ -131,3 +103,7 @@ parts:
          "$SNAPCRAFT_PART_INSTALL/usr/share/doc/device-simple/Attribution.txt"
       install -DT "./LICENSE" \
          "$SNAPCRAFT_PART_INSTALL/usr/share/doc/device-simple/LICENSE"
+
+  config-common:
```

Comment:
This stanza can be removed as there aren't any runtime-helpers included in this snap.